### DISCUSSION TITLE
can the astro file which specific locale be used preferentially

### DISCUSSION THREAD

**Discussion by minzojian** [jlarmstrongiv/astro-i18n-aut#50] [Q&A] [answered]
``astro-i18n-aut`` can avoids template file duplication, that's cool. however, if i have some page that is under some specfic locale, can it be used preferentially?

for example, my project structure is like this:

-- pages
----index.astro
----about.astro
----fr
------index.astro

the default locale is en, and `prefixDefaultLocale: false`
suppose the homepage of fr is total different than the default one,
that i have to create a sperate page `pages/fr/index.astro`
in my thought,i prefer use that file to render the homepage of fr, other than use the default homepage file of en.
however, it is not support right now, and when build page, it will say
``An integration attempted to inject a route that is already used in your project: "/en" at "src/pages/index.astro". This route collides with: "src/pages/fr/index.astro".``

can this feature be implemented?

**Comment by jlarmstrongiv** [comment, marked as answer]
Hey @minzojian,

Have you tried switching out your components inside the `index.astro` file depending on the language?  That’s how I would recommend customizing your pages depending on the language.

Or, have you tried excluding that page `pages/index.astro` with `exclude`?  That would mean you would have to make a custom page for each language though
